docs/docsite/rst/installation_guide/installation_distros.rst

```diff
@@ -7,23 +7,21 @@ Installing Ansible on specific operating systems
 
 The ``ansible`` package can always be :ref:`installed from PyPI using pip <intro_installation_guide>` on most systems but it is also packaged and maintained by the community for a variety of Linux distributions.
 
-The following instructions will guide you through installing the ``ansible`` package with your preferred distribution's package manager.
+This document guides you through installing the ``ansible`` package with different package managers.
 
-.. note:: For maintainers who wish to add distributions to this guide, installation instructions are included here only for distributions with a reasonably up-to-date version of ``ansible``. The distribution MUST ensure that ``ansible-core`` and ``ansible`` versions are kept in sync to the extent that the distribution build system allows. Maintainers MUST include a way to contact them with their instructions here and are encouraged to join the `Ansible Packaging <https://matrix.to/#/#packaging:ansible.com>`_ Matrix room.
+To add instructions for another distribution to this guide, maintainers for that distribution **must** do the following:
+
+    * Ensure the distribution provides a reasonably up-to-date version of ``ansible``.
+    * Ensure that ``ansible-core`` and ``ansible`` versions are kept synchronized as closely as possible, to the extent that the build system allows.
+    * Provide a way to contact the distribution maintainers as part of the instructions. Distribution maintainers are also encouraged to join the `Ansible Packaging <https://matrix.to/#/#packaging:ansible.com>`_ Matrix room.
 
 .. contents::
   :local:
 
 Installing Ansible on Fedora Linux
--------------------------------------------------
-
-To install the batteries included ``ansible`` package on Fedora run
-
-.. code-block:: bash
-
-    $ sudo dnf install ansible
+----------------------------------
 
-If you prefer to install the minimal ``ansible-core`` package run
+To install the minimal ``ansible-core`` package on Fedora run
 
```

Comment:
I commented on another PR but will repeat here - We chose specifically to list ansible package installs first as that is the community's preferred installation. If you want to change this, please open an issue so the community can discuss  the pros/cons of changing the order to put core install first.

Reply:
> I commented on another PR but will repeat here - We chose specifically to list ansible package installs first as that is the community's preferred installation. If you want to change this, please open an issue so the community can discuss the pros/cons of changing the order to put core install first.

I tend to agree.

Reply:
I'll mark this as a draft so we can discuss that point in more detail.